deploy/aws/cloudbeat-aws.yml

```diff
@@ -0,0 +1,66 @@
+cloudbeat:
+  type: cloudbeat/cis_aws
+  access_key_id: ""
+  secret_access_key: ""
+  # Defines how often an event is sent to the output
+  period: 30s
+  fetchers:
+    - name: aws-iam
+  evaluator:
+    decision_logs: false
+  runtime_cfg:
+    activated_rules:
+      cis_aws:
+        - cis_1_8
+        - cis_1_9
+# =================================== Kibana ===================================
+setup.kibana:
+
+  # Kibana Host
+  host: "http://localhost:5601"
```

Comment:
```suggestion
  host: "http://host.docker.internal:5601"
```

Reply:
CSPM deployment won't be k8s related, that's why I thought it shouldn't reside under the `kustomize/overlays/` dir.